test/active_record_test.rb

```diff
@@ -7,6 +7,7 @@
 #@formatter:off
 ActiveRecord::Schema.define version: 0 do
   create_table :users, force: true do |t|
+    t.string :bank_account_number
```

Comment:
Good point. The thing is, if we don't add the plaintext attribute to the database, we miss out on Active Record picking up the attribute. 

I think this line from the Active Record source helps explain:

 > Active Record objects don't specify their attributes directly, but rather infer them from the table definition with which they're linked.
>
> Source: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/base.rb#L27

I needed a full Active Record attribute to test the fix. (I chose `bank_account_number` mainly because it wasn't in use by other tests.)

I agree it's not exactly optimal to place that lonely attribute in the test table like that, heh. I didn't have a better approach at the time. I'm open to any suggestions on how to get something like `bank_account_number` treated as a full Active Record attribute though.